### DISCUSSION TITLE
New collection management logic in upstream

### DISCUSSION THREAD

**Discussion by jfstenuit** [dani-garcia/vaultwarden#4527] [General]
It seems upstream has impltion management logic.

When managing roles, Vaultwarden still has the "User,Manager,Admin,Owner" roles available, but when clicking on the help (?) icon, we are redirected to upstream documentation where only "User,Admin,Owner" roles are defined.

Apparently, the change dates back from upstream's 2024.2.2 version.

What is the plan here ?
- adapt vaultwarden to match upstream's new logic
- adapt documentation so that the (?) points to an updated vaultwarden wiki page

**Comment by BlackDex** [comment]
We need to migrate to the new flexibele way of management for this. The old way will be removed if I'm correct.

We have not yet started on implementing this.
It also needs a migration step, so that is going to be tricky.
